config/304-xslttransform.yaml

```diff
@@ -98,7 +98,8 @@ spec:
                 type: object
                 properties:
                   ref:
-                    description: Reference to an addressable Kubernetes object to be used as the destination of events. If left empty, the events will be sent back to the sender/source. 
+                    description: -|
```

Comment:
Tip: You simply need a carriage return before the 120 characters mark to keep the linter happy. No need for `: |` or any other alternative syntax.

Reply:
oh nice! thank you!